test/abs_form_test.ml

```diff
@@ -212,7 +212,7 @@ let%expect_test "test04.beam" =
             (DeclType 6 tuple
               ((6 A)
                (6 B))
-              (TyPredef 6 tuple (
+              (TyTuple 6 (
```

Comment:
In [absform document](http://erlang.org/doc/apps/erts/absform.html), tuple type is not a predefined type.

Reply:
:+1: